internal/venice-common/src/main/java/com/linkedin/venice/meta/AbstractStore.java

```diff
@@ -107,7 +107,11 @@ public void checkDisableStoreWrite(String action, int version) {
    *                        any store level config on it; if false, the version being added is new version, so the new version
    *                        config should be the same as store config.
    */
-  private void addVersion(Version version, boolean checkDisableWrite, boolean isClonedVersion) {
+  private void addVersion(
+      Version version,
+      boolean checkDisableWrite,
+      boolean isClonedVersion,
+      int currentRTVersionNumber) {
```

Comment:
not using `currentRTVersionNumber` in the method yet. That will be the part of future PR#1555
because the logic is closer to that PR's work.
Some changes in this PR are going to make that long PR more reviewable.